Header header
float32 charge
float32 capacity
